index.html

```diff
@@ -0,0 +1,139 @@
+            <li class="list"><a href="#form">Join</a></li>
+            <li class="list"><a href="#contact">Contact us</a></li>
+        </ul>
+       
+    </div>
+
+    <div>
+      <a class ="nav-image" href="./about.html">
+      <img src="./images/240957a3-789c-4e2b-9fc7-34a08b3e9f0c.png" class="crypto-image" alt="crypto-logo" >
+      </a>
+    </div>
+
+    <nav class="navigation">
+       
+        <div class="nav-text">
+        <div class="about"> <a href="./about.html"> About</a></div>
+        <div class="home"> <a href="./index.html">Home</a></div>
+        <div class="join"><a href="#form">Join</a></div>
+        <div class="contact" > <a href="#contact">Contact us </a></div>
+        </div>
+    </nav>
+  
+
+    <section class="intro-section">
+
+      <article class="intro-align">
+    <div class="crypto-text" id="intro">"Blockchain crypto!"</div>
+
+    
+    <h1 class="ico-title">NEW CRYPTO COIN 2022 INITIAL COIN OFFERING</h1>
+    <div class="ico-words"> The crypto currency that has come to create financial freedom and be the foundation of the future of finance,all over the world. The ico will be taking place on December .</div>
+    <h2 class="ico-date">2022.12.01(TUE)~30(FRI)</h2>
+    <div id="contact" class="contact">@ www.fakenewcryptocoin.com</div>
+      </article>
+
+    </section>
+
+
+    <section class="main-section">
+        
+    <h2 class="program-class">Main Program</h2>
+        
+        <div class="card-flex">
+            
+      <div class="card">
+        <img src="./images/download.jpeg" alt="whitepaper image" class="program-img">
+        <div class="card-container">
+          <h4><b>Release of official complete white paper</b></h4>
+        </div>
+
+      </div>
+
+      <div class="card">
+        <img src="./images/download.png" alt="explanation" class="program-img">
+        <div class="card-container">
+          <h4><b>Explanation of whitepaper</b></h4>
+        </div>
+      </div>
+      
+        <div class="card">
+            <img src="./images/download (1).jpeg" alt="roadmap" class="program-img">
+            <div class="card-container">
+              <h4><b>official roadmap</b></h4>
+            </div>
+        
+        </div>
+      </div>
+        </div>
+        
+        <form action="https://formspree.io/f/xbjbawkn" method="post" id="form">
+            <input type="text" name="Name" maxlength="30" placeholder="Name">
+    
+            <div class="submit">
+            Please type/text your email here to receive authentication email to participate in the ICO.
+            <textarea name="e-mail" cols="40" rows="1" placeholder="E-mail"></textarea>
+            <button type="submit" value="Submit">Submit</button>
+          </div>
+        </form> 
+    </section>
+
+    
+
+    <section class="speaker-section">
+        
+        <ul class="speakers-list">
+    <h2>Featured Speakers</h2>
+    <hr class="speaker-line1">
+
+    <div class="list-container">
+    
+       
+    </div>
```

Comment:
Notice that the description text is flowing down and overlapping the next speaker's div. Kindly try to design it closely to the design given.
| Your Design   | Behance Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219802942-0044d975-dd2f-4b30-8a9f-dc941d6176db.png)| ![image](https://user-images.githubusercontent.com/61361037/219803020-82efe4eb-6b23-4f5f-b13a-aa567ec83220.png)|